xml/System.Windows.Automation.Provider/ITextProvider.xml

```diff
@@ -235,7 +235,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets a value that specifies whether a text provider supports selection and, if so, the type of selection supported.</summary>
-        <value>One of <see langword="None" />, <see langword="Single" />, or <see langword="Multiple" /> from <see cref="T:System.Windows.Automation.SupportedTextSelection" />.</value>
+        <value>One of <see langword="None" />, <see cref="T:System.Single" />, or <see langword="Multiple" /> from <see cref="T:System.Windows.Automation.SupportedTextSelection" />.</value>
         <remarks>To be added.</remarks>
```

Comment:
I'm not convinced the `Single` mentioned here is referring to the `T:System.Single`.

Reply:
Updated. I don't think it's necessary to list all possible enum values.